cmd/cloud_login.go

```diff
@@ -115,6 +134,26 @@ func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
 		newCloudConf.Token = null.StringFrom(vals["Token"])
 	}
 
+	if newCloudConf.Token.Valid {
+		client := cloudapi.NewClient(
+			c.globalState.Logger,
+			newCloudConf.Token.String,
+			consolidatedCurrentConfig.Host.String,
+			consts.Version,
+			consolidatedCurrentConfig.Timeout.TimeDuration(),
+		)
+
+		var res *cloudapi.ValidateTokenResponse
+		res, err = client.ValidateToken()
+		if err != nil {
+			return err
+		}
+
+		if !res.IsValid {
+			return errors.New(`your API token is invalid, please generate a new one at https://app.k6.io/account/api-token`)
```

Comment:
Don't we have a grafana url? as we are probably going to have to drop this either way soon.